ush/python/pygfs/task/land_analysis.py

```diff
@@ -180,3 +320,168 @@ def get_bkg_dict(self, config: Dict) -> Dict[str, List[str]]:
+    @logit(logger)
+    def get_ens_bkg_dict(config: Dict) -> Dict:
+        """Compile a dictionary of model background files to copy for the ensemble
+        Note that a "Fake" 2-member ensemble backgroud is being created by copying FV3 RESTART files (coupler, sfc_data)
+        from the deterministic background to DATA/bkg/mem001, 002.
+
+         Parameters
+         ----------
+         config: Dict
+             Dictionary of key-value pairs needed in this method
+             Should contain the following keys:
+             COM_ATMOS_RESTART_PREV
+             DATA
+             current_cycle
+             ntiles
+
+         Returns
+         ----------
+         bkg_dict: Dict
+             a dictionary containing the list of model background files to copy for FileHandler
+         """
+
+        dirlist = []
+        bkglist = []
+
+        # get FV3 sfc_data RESTART files; Note an ensemble is being created
+        rst_dir = os.path.join(config.COM_ATMOS_RESTART_PREV)
+
+        for imem in range(1, LandAnalysis.NMEM_LANDENS + 1):
+            memchar = f"mem{imem:03d}"
+
+            run_dir = os.path.join(config.DATA, 'bkg', memchar, 'RESTART')
+            dirlist.append(run_dir)
+
+            # Land DA needs coupler
+            basename = f'{to_fv3time(config.current_cycle)}.coupler.res'
+            bkglist.append([os.path.join(rst_dir, basename), os.path.join(run_dir, basename)])
+
+            # Land DA only needs sfc_data
+            for ftype in ['sfc_data']:
+                template = f'{to_fv3time(config.current_cycle)}.{ftype}.tile{{tilenum}}.nc'
+                for itile in range(1, config.ntiles + 1):
+                    basename = template.format(tilenum=itile)
+                    bkglist.append([os.path.join(rst_dir, basename), os.path.join(run_dir, basename)])
+
+        bkg_dict = {
+            'mkdir': dirlist,
+            'copy': bkglist
+        }
+
+        return bkg_dict
+
+    @staticmethod
+    @logit(logger)
+    def create_ensemble(vname: str, bestddev: float, config: Dict) -> None:
+        """Create an ensemble for Snow Depth analysis by perturbing snow depth with a prescribed variance.
+        Additionally, remove glacier locations
+
+        Parameters
+        ----------
+        vname : str
+            snow depth variable to perturb. "snowdl" or "snwdph" depending on FRACGRID (YES or NO)
+        bestddev : float
+            Background Error Standard Deviation to perturb around to create ensemble
+        config: Dict
+            Dictionary of key-value pairs needed in this method.  It must contain the following keys:
+            DATA
+            current_cycle
+            ntiles
+        """
+
+        # 2 ens members
+        offset = bestddev / np.sqrt(LandAnalysis.NMEM_LANDENS)
+
+        logger.info(f"Creating ensemble for LETKFOI by offsetting with {offset}")
+
+        workdir = os.path.join(config.DATA, 'bkg')
+
+        sign = [1, -1]
+        ens_dirs = ['mem001', 'mem002']
```

Comment:
There should be some sort of comment here that these are currently hard-coded for a 2-mem ensemble.

Reply:
The doc-block says, a Fake 2 member ensemble.  Isn't that sufficient?

Reply:
It's not in the doc-block for this method, and either way it is nice to identify which parts need to be replaced later and which parts don't. But I'll leave it up to you.